src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
You can **separate creation** of these 3 objects **and their printing** in a console and add emty line between them for better readability. <br/> <br/> *(additional) you can also think about reducing the amount of the code using **Stream API**
```suggestion
        Lottery ball1 = new Lottery();
        Lottery ball2 = new Lottery();
        Lottery ball3 = new Lottery();
        
        System.out.println(ball1.getRandomBall().toString());
        System.out.println(ball2.getRandomBall().toString());
        System.out.println(ball3.getRandomBall().toString());
```